lua/md-pdf/config.lua

```diff
@@ -0,0 +1,28 @@
+local uv = vim.loop
+
+local M = {}
+
+local defaults = {
+	margins = "1.5cm",
+	--- tango, pygments are quite nice for white on white
+	highlight = "tango",
+	--- Generate a table of contents, on by default
+	toc = true,
+	--- The command to open the pdf with
+	--- @type string | function
+	preview_cmd = function()
+		local os_used = uv.os_uname().sysname
+		if os_used == 'Linux' then return 'xdg-open' end
+		if os_used == 'Darwin' then return 'open' end
+		-- assume the other OS is windows for now
```

Comment:
Would appreciate if you could reinsert the todo for BSD:
`-- TODO: (aver) Add BSD check`